webui/react/src/e2e/tests/userManagement.spec.ts

```diff
@@ -19,6 +19,35 @@ test.describe('User Management', () => {
     await expect(page).toHaveURL(userManagementPage.url);
   });
 
+  test('Users table count matches admin page users tab', async ({ page }) => {
+    const userManagementPage = new UserManagement(page);
+    await userManagementPage.goto();
+    const pagination = userManagementPage.table.table.pagination;
+    const match = (await userManagementPage.userTab.pwLocator.innerText()).match(/\d+/);
+    if (match === null) {
+      throw new Error('Number not present in tab.');
+    }
+    const expetedRowCount = +match[0];
+    for (const paginationOption of [
+      pagination.perPage.perPage10,
+      pagination.perPage.perPage20,
+      pagination.perPage.perPage50,
+      pagination.perPage.perPage100,
+    ]) {
+      await pagination.perPage.pwLocator.click();
```

Comment:
could we make each of these a test.step() so we can see on failure which one failed quickly? IDK if playwright lets us put steps and for loops or not, but could be nice to have.